ols/src/prompts/prompts.py

```diff
@@ -6,18 +6,49 @@
 from ols.constants import SUBJECT_ALLOWED, SUBJECT_REJECTED
 
 # TODO: OLS-503 Fine tune system prompt
-QUERY_SYSTEM_PROMPT = """You are an assistant for question-answering tasks \
-related to the openshift and kubernetes container orchestration platforms. \
+
+# Note::
+# Right now templates are somewhat alligned to make granite work better.
+# GPT still works well with this. Ideally we should have model specific tags.
+# For history we can laverage ChatPromptTemplate from langchain,
+# but that is not done as granite was adding role tags like `Human:` in the response.
+# With PromptTemplate, we have more control how we want to structure the prompt.
+
+QUERY_SYSTEM_INSTRUCTION = """
+You are an assistant for question-answering tasks \
+related to the openshift and kubernetes container orchestration platforms.
```

Comment:
these system instructions seem a bit brief....is there more we should consider adding here?  telling the model to be polite, not to make up answers, only give answers it's confident in, explicitly tell it not to try to answer non-openshift questions, etc?

(i know you just copied the existing instructions, but we've never really played w/ this at all)